src/charactersheet/models/character/ability_score.js

```diff
@@ -6,6 +6,7 @@ import ko from 'knockout';
 
 export class AbilityScore extends KOModel {
     static __skeys__ = ['core', 'abilityScores'];
+    static __dependents__ = ['Skill', 'SavingThrow'];
```

Comment:
I forgot the rules to the relationship, but shouldn't more things be in this list? Like weapons, armor, other stats, etc.

Reply:
While Weapons and Armor use data from the Ability Scores, they do so by fetching the scores directly, not via a nested relationship in the API response. 

Hypnos' dependents tracking applies only when updating a resource would cause the data from another resource to be out of date. Since Armor receives a local signal, and then re-fetches the Ability Scores to compute the value, this is not needed.